lib/community.js

```diff
@@ -451,6 +451,14 @@ exports.all = {
         quote: 'hapi has been a solid, extensible, test and performance friendly server framework that’s continually evolving and improving. It allows us to build and maintain secure and stable services quickly that scale quite well. I never thought I’d say this but, thanks Walmart!',
         person: 'Nathan Heskew, Developer'
     },
+    'Teachoo': {
+        url: 'https://www.teachoo.com/',
+        logo: 'logo-teachoo.svg',
```

Comment:
I think it should be logo-teachoo.**png** and not logo-teachoo.svg considering the file you added to your PR.